mlrun/feature_store/api.py

```diff
@@ -125,30 +123,41 @@ def get_offline_features(
-    :param drop_columns:   list of columns to drop from the final result
+    :param feature_vector:          feature vector uri or FeatureVector object. passing feature vector obj requires
+                                    update permissions
+    :param entity_rows:             dataframe with entity rows to join with
+    :param target:                  where to write the results to
+    :param drop_columns:            list of columns to drop from the final result
     :param entity_timestamp_column: timestamp column name in the entity rows dataframe
-    :param run_config:     function and/or run configuration
-                           see :py:class:`~mlrun.feature_store.RunConfig`
-    :param start_time:      datetime, low limit of time needed to be filtered. Optional.
-        entity_timestamp_column must be passed when using time filtering.
-    :param end_time:        datetime, high limit of time needed to be filtered. Optional.
-        entity_timestamp_column must be passed when using time filtering.
-    :param with_indexes:    return vector with index columns and timestamp_key from the feature sets (default False)
-    :param update_stats:    update features statistics from the requested feature sets on the vector. Default is False.
-    :param engine:          processing engine kind ("local", "dask", or "spark")
-    :param engine_args:     kwargs for the processing engine
-    :param query:           The query string used to filter rows
-    :param spark_service:   Name of the spark service to be used (when using a remote-spark runtime)
+
+    :param run_config:              function and/or run configuration
+                                    see :py:class:`~mlrun.feature_store.RunConfig`
+    :param start_time:              datetime, low limit of time needed to be filtered. Optional.
+                                    entity_timestamp_column must be passed when using time filtering.
+    :param end_time:                datetime, high limit of time needed to be filtered. Optional.
+                                    entity_timestamp_column must be passed when using time filtering.
+    :param with_indexes:            return vector with index columns and timestamp_key from the feature sets
+                                    (default False)
+    :param update_stats:            update features statistics from the requested feature sets on the vector.
+                                    Default is False.
+    :param engine:                  processing engine kind ("local", "dask", or "spark")
+    :param engine_args:             kwargs for the processing engine
+    :param query:                   The query string used to filter rows
+    :param join_type:               {'left', 'right', 'outer', 'inner'}, default 'inner'
+                                    Only for dask and local engines (not for spark)
+                                    Used for regular join when timestemp and feature_vector.spec.timestamp_field
```

Comment:
this line is not clear to me. please elaborate.

Reply:
This parameter is in use when entity_timestamp_column and
                                    feature_vector.spec.timestamp_field are None, if one of them 
                                    isn't none we're preforming as_of join.